tsconfig.json

```diff
@@ -2,7 +2,13 @@
   "compilerOptions": {
     "target": "esnext",
     "module": "esnext",
-    "strict": true,
```

Comment:
Why the change here? Is there a specific property that you didn't want strict that was covered in the list?

Reply:
As you know, `strict` covers all the other properties. I needed to disable a couple of properties specifically (due to SbcFeeSummary) but I also wanted to keep the other checks enabled.

Reply:
If you feel this doesn't compromise our goal of being strict and won't carry over into other elements of development, i'm 100% fine with it. Just at first glance, those 3 options certainly do lessen the rules. Is this the only way?

Reply:
Discussed with Severin that this is a temporary change until we can fix some type issues with sbc common components. Resolved!

Reply:
I have created ticket /bcgov/entity#2357 to get the SbcFeeSummary updated so that we can go back to `strict: true`.